app/assets/javascripts/factories/repository/repository_data.js

```diff
@@ -0,0 +1,18 @@
+ManageIQ.angularApplication.factory('repositoryDataFactory', ['$http', function($http) {
+    var factoryRepository = {
```

Comment:
@himdel I don't know ... using a ```Factory``` seemed like an obvious choice to me, maybe because of some of the examples that I've been looking at. 
Isn't SSUI also using factories ?